pkg/os/volume/api.go

```diff
@@ -197,8 +197,8 @@ func (VolumeAPI) ResizeVolume(volumeID string, size int64) error {
 		return fmt.Errorf("error getting the current size of volume (%s) with error (%v)", volumeID, err)
 	}
 
-	//if the partition's size is already the size we want this is a noop, just return
-	if currentSize >= finalSize {
+	// only resize if finalSize - currentSize is greater than 100MB
+	if finalSize-currentSize < 100*1024*1024 {
```

Comment:
Is 1GB the minimal quantity allowed for resize. I think this would depend on the provider right?

Reply:
ah I think I misread it, my bad.